test/ScanDependencies/module_deps.swift

```diff
@@ -133,7 +133,6 @@ import SubE
 // CHECK: "commandLine": [
 // CHECK: "-compile-module-from-interface"
 // CHECK: "-target"
-// CHECK: "-sdk"
```

Comment:
Should we add some `CHECK-NOT`s here to make sure there aren't any of the `-I`'s here?

Reply:
Yeah, will add.